packages/osd-apm-config-loader/src/utils/read_config.ts

```diff
@@ -29,13 +29,13 @@
  */
 
 import { readFileSync } from 'fs';
-import { safeLoad } from 'js-yaml';
+import { load } from 'js-yaml';
```

Comment:
@andreymyssak Can you explain this change a little? Is it just a method renaming, or are there functional differences between the behavior of the old `safeLoad` vs `load`?

Reply:
Just renaming, see [safeLoad(), safeLoadAll(), safeDump() => load(), loadAll(), dump()](https://github.com/nodeca/js-yaml/blob/master/migrate_v3_to_v4.md#safeload-safeloadall-safedump--load-loadall-dump). There's no mention of a breaking change in their behavior.

Reply:
perfect, thanks for the link!